### DISCUSSION TITLE
Where to download midori browser for mac user?

### DISCUSSION THREAD

**Discussion by flixyudh** [goastian/midori-desktop#73] [Q&A]
as the `readme` state:

> Midori Browser is available for Windows, macOS, and Linux. You can install it by running the installer or by extracting the archive.

I Expecting midori already support for mac, but I don't see any `dmg` or `pkg` file in https://github.com/goastian/midori-desktop/releases and the official website telling that macOS still `comming soon`, I'm confused is midori already support macOS or not?

**Comment by Ponchale** [comment]
> as the `readme` state:
> 
> > Midori Browser is available for Windows, macOS, and Linux. You can install it by running the installer or by extracting the archive.
> 
> I Expecting midori already support for mac, but I don't see any `dmg` or `pkg` file in https://github.com/goastian/midori-desktop/releases and the official website telling that macOS still `comming soon`, I'm confused is midori already support macOS or not?

We don't have a MacOS version yet but we will have it very soon.
